cmd/cri-containerd/options/options.go

```diff
@@ -63,12 +64,24 @@ type CniConfig struct {
 	NetworkPluginConfDir string `toml:"conf_dir" json:"confDir,omitempty"`
 }
 
+// Mirror contains the config related to the registry mirror
+type Mirror struct {
+	Endpoint string `toml:"endpoint" json:"mirror,omitempty"`
+}
+
+// Registry is registry settings configured
+type Registry struct {
+	Mirrors []Mirror
```

Comment:
Can we give this field a toml name?
Do `cri-containerd default-config`, you will know why I ask for that. :)

Reply:
will do